rdt/transformers/numerical.py

```diff
@@ -546,8 +548,6 @@ def _reverse_transform(self, data):
 
         recovered_data = self._reverse_transform_helper(data)
         if self.null_transformer and self.null_transformer.models_missing_values():
-            data = np.stack([recovered_data, data[:, -1]], axis=1)  # noqa: PD013
-        else:
-            data = recovered_data
+            recovered_data = np.stack([recovered_data, data[:, -1]], axis=1)  # noqa: PD013
```

Comment:
Can we get rid of this noqa?

Reply:
Nope. It thinks that is pandas `stack` and tells you to use `melt`, which is a method numpy doesn't have.

Reply:
Is `np.concatenate` the same ?

Reply:
`stack` concatenates them along a different axis. It's also supposed to be faster. I think we could change it to `np.concatenate([[array1], [array2]], axis=1)`, or something like that, but that just seemed more confusing than the `noqa`.